src/Solver.cpp

```diff
@@ -298,6 +298,17 @@ FP_PRECISION Solver::getFlux(int fsr_id, int group) {
 
 
 /**
+ * @brief Returns the boundary flux array at the requested indexes
```

Comment:
So are you going to change it to indices or "flux array for some track". Your link seems to suggest indices is the way to go for this context.